federatedscope/core/auxiliaries/splitter_builder.py

```diff
@@ -0,0 +1,45 @@
+import logging
+import federatedscope.register as register
+
+logger = logging.getLogger(__name__)
+
+
+def get_splitter(config):
+    client_num = config.federate.client_num
+    if config.data.splitter_args:
+        args = config.data.splitter_args[0]
+    else:
+        args = {}
+
+    for func in register.splitter_dict.values():
+        splitter = func(config)
+        if splitter is not None:
+            return splitter
+    # Delay import
+    # generic splitter
+    if config.data.splitter == 'lda':
+        from federatedscope.core.splitters.generic import LDASplitter
+        splitter = LDASplitter(client_num, **args)
+    # graph splitter
```

Comment:
Is it possible to make these graph splitters applicable to other data formats? That is, they can use the same splitting algorithms but with a bit different pre/post processes.

Reply:
Sure, I will add and modify these splitters in the next PR.